daily.php

```diff
@@ -90,6 +90,11 @@
     }
 }
 
+if ($options['f'] === 'ports_fdb') {
+    $ret = lock_and_purge('ports_fdb', 'updated_at < DATE_SUB(NOW(), INTERVAL ? DAY)');
```

Comment:
You need to set this default value and add documentation

Reply:
default value is already set here : 
includes/defaults.inc.php

I'll do the documentation if you are OK with current implementation.

Reply:
Doc was updated